docs/storage.md

```diff
@@ -493,6 +494,72 @@ config:
 prefix: ""
 ```
 
+### Oracle Cloud Infrastructure Object Storage
+
+To configure Oracle Cloud Infrastructure (OCI) Object Storage as Thanos Object Store, you need to provide appropriate authentication credentials to your OCI tenancy. The OCI object storage client implementation for Thanos supports either the default keypair or instance principal authentication.
+
+#### API Signing Key
+
+The default API signing key authentication provider leverages same [configuration as the OCI CLI](https://docs.oracle.com/en-us/iaas/Content/API/Concepts/cliconcepts.htm) which is usually stored in at `$HOME/.oci/config` or via variable names starting with the string `OCI_CLI`. You can also use environment variables that start with `TF_VAR`. If the same configuration is found in multiple places the provider will prefer the first one.
```

Comment:
> You can also use environment variables that start with `TF_VAR`.

This sounds funny to me. What does Thanos's OCI storage implementation have to do with Teratogen
Terraform?

Reply:
@squat 
Where did you see Teratogen? Can you clarify? Thx!

Reply:
I'll assume @squat had an autocorrect mishap, and he meant simply Terraform :), I assume he refers to why we're mentioning the `TF_VAR` here.

Reply:
Thanks @squat and @matej-g for the eagle eyes.
Updated my PR. Please review again.